test/runtests.jl

```diff
@@ -699,3 +699,34 @@ end
     U = transform(t, x)
     @test isfinite(logabsdet(U)[1])
 end
+
+@testset "inverse_eltype of scalar transforms with parameters" begin
+    # `Float64` parameters and `Float32` input
+    for t in (as(Real, 0.5, ∞), as(Real, -∞, 2.1), as(Real, 0.5, 2.1))
+        @test @inferred(inverse_eltype(t, 1.1f0)) === Float64
+        @test inverse(t, 1.1f0) isa Float64
+    end
+
+    # Derivatives wrt parameters of the transforms
+    d1 = ForwardDiff.derivative(5.3) do x
+        return only(inverse(as(Vector, as(Real, x, ∞), 1), [10]))
```

Comment:
Would it make sense to add `@inferred` here, eg
```suggestion
        return @inferred only(inverse(as(Vector, as(Real, x, ∞), 1), [10]))
```
and below?